homework01/vector.h

```diff
@@ -1,5 +1,35 @@
 
 class Vector
 {
-	// TODO
+ public:
+  Vector(double x = 0., double y = 0.)
+    {
+      this->x = x;
+      this->y = y;
+    }
+  
+  bool operator==(Vector other)
```

Comment:
We didn't discuss this yet, but as this method is not modifying the object state, it must be marked as `const`: https://en.cppreference.com/w/cpp/language/member_functions#const-.2C_volatile-.2C_and_ref-qualified_member_functions